internal/tools/orchestrator/services/addon/addon_handler.go

```diff
@@ -127,7 +127,12 @@ func (a *Addon) GetAddonExtention(params *apistructs.AddonHandlerCreateItem) (*a
 		if emptyVersion {
 			err = errors.New(i18n.OrgSprintf(params.OrgID, AddonDefaultVersionDoseNoExist, params.AddonName))
 		} else {
-			err = errors.New(i18n.OrgSprintf(params.OrgID, AddonVersionDoseNoExist, params.AddonName, version))
+			def, ok := addons.GetDefault()
```

Comment:
Clearly the main purpose of this code snippet is to obtain the recommendVersion:

```golang
recommendVersion := "-"
if def, ok := addons.GetDefault(); ok {
  recommendVersion = def.Version
}
```